docs/source/GUI.rst

```diff
@@ -13,23 +13,23 @@ how use it, first load and run either \* demo_OnACID_mesoscope.py \*
 demo_caiman_basic.py \* demo_pipeline.py
 
 Each of these demos will save a results file. You can then start the
-visual interface by running the following command from the base caiman
-folder (make sure you are within your caiman environment):
+visual interface by visiting the caiman_data folder, under the bin folder,
+and issuing the following command:
 
 ::
 
-   ipython caiman/gui/gui_pyqtgraph_layout.py
+   python caiman_gui.py
```

Comment:
same question as earlier: `python` or `ipython`?
@pgunn

Reply:
I tested it with regular python and it behaved fine. It doesn't seem to depend on ipython.